src/page-header/base/page-header.component.ts

```diff
@@ -68,7 +68,7 @@ export const itemsWithTitle = (items: BreadcrumbItem[], title: string): Breadcru
 				</ibm-breadcrumb>
 				<div class="ibm--page-header__title-container">
 					<h1 class="ibm--page-header__title" [attr.title]="title">
-						{{ title }}
+						{{ title }} <ng-content select="[title]"></ng-content>
```

Comment:
Would it still work if we do
```suggestion
						{{ title }}
						<ng-content select="[title]"></ng-content>
```
?

Reply:
yes